includes/text2dna.h

```diff
@@ -0,0 +1,15 @@
+// A set of useful headers.
+
+#ifndef TEXT2DNA_H
+#define TEXT2DNA_H
+
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#define ASCII_BITS_MAXLEN 7 // E.G. 127 is 111111 (7 digits)
```

Comment:
We used this macro constant to decide how much to malloc given the max ASCII value we will have to deal with is 127, which is 1333 in base 4. This is four digits so we should change this constant to 4. We should also rename this to be consistent, like `ASCII_BASE7_MAXLEN`.